package.json

```diff
@@ -46,6 +46,9 @@
     "ext": "js,html,njk,scss"
   },
   "standard": {
+    "env":  [
```

Comment:
to fix this 
![image](https://user-images.githubusercontent.com/98027245/174828781-04634251-0366-4386-bf30-8447ae9e8520.png)

Reply:
I've not seen this before, does this achieve the same thing as declaring globals [like so](https://github.com/ministryofjustice/prepare-a-case/blob/53c49d73351fbd6b0d32c61dc1ef5e01dc15de16/tests/services/community-service.test.js#L1) but globally?

If so is there a reason why we've not done this previously? Unlikely to happen but will this cause standard to accept `describe/it/etc` in production code, i.e. not in tests? @theDustRoom @sarahyjja any thoughts?

Reply:
Yes, as the `describe/it/etc` are only ever in use within our tests we declare them at the start of the tests so that standard is happy and does not include these globals elsewhere when they're not needed.

Reply:
Cool, that makes sense. Probably best to go with something like `/* global describe, beforeEach, afterEach, it, expect, jest */` in the new test then and revert this if that's ok @rschallapalli?